_data/blog_posts.yml

```diff
@@ -1,3 +1,12 @@
+- title: "A talk on the new AWS Application Load Balancer, updates to ECS, and Kinesis Analytics"
+  url: "https://blog.gruntwork.io/a-talk-on-the-new-aws-application-load-balancer-updates-to-ecs-and-kinesis-analytics-abb599cb3cb8#.vhbek5ce2"
+  thumbnail_url: "https://i.imgur.com/KkniqPD.png"
```

Comment:
Actually, you *might* be able to use the SlideShare deck itself as a thumbnail: https://help.medium.com/hc/en-us/articles/215680047-Featured-image

Select it in the post and hit `CMD+OPTION+8`. Not exactly intuitive or discoverable, is it?